src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
-            <h2>Apply for funding</h2>
-            <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an application</a>
-            <p></p>
-            <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on seminars</a>
-            <p></p>
+            <h2 class="heading-large">Apply for funding</h2>
+
+            <p>
+                <a href="@controllers.routes.ApplicationController.showOrCreateForForm(applicationId)">Make an
+                    application</a>
+            </p>
+
+            <p>
+                <a href="@controllers.routes.OpportunityController.showGuidancePage(opportunity.id)">Guidance on
+                    seminars</a>
+            </p>
         </div>
 
         @currentSection.map { s =>
-            <article>
-                <h1>@s.sectionNumber. @s.title</h1>
-                @if(sectionNumber == 1) {
-                    <div>
-                        <h3>Start date</h3>
-                        @opportunity.startDate
-                        <h3>Days duration</h3>
-                        @opportunity.duration.map(d => s"${d.duration} ${d.units}").getOrElse("Open indefinitely")
-                        <h3>Opportunity value</h3>
-                            &pound; @opportunity.value.amount @opportunity.value.unit
-                    </div>
-                    <p></p>
-                }
-                @s.paragraphs.map { p =>
-                    <p>@p</p>
-                }
-            </article>
-            <footer>
+
+            @if(sectionNumber == 1) {
+
+        <div class="panel panel-border-narrow">
+            <aside role="complementary" class="notice">
+                <div class="column-half">
+                    <dl>
+                        <dt class="heading-small">Deadline</dt>
+                        <dd>Apply any time</dd>
+                    </dl>
+                </div>
+
+                <div class="column-half">
+                    <dl>
+                        <dt class="heading-small">Value</dt>
+                        <dd>&pound;@opportunity.value.amount @opportunity.value.unit</dd>
+                    </dl>
+                </div>
+            </aside>
+        </div>
+
+            }
+        <article>
+            <h2 class="heading-large">@s.sectionNumber. @s.title</h2>
+            @s.paragraphs.map { p =>
+                <p class="text">@p</p>
+            }
+        </article>
+
+        <footer class="contain-floats">
+            <nav role="navigation" class="page-navigation">
                 @prevSection.map { s =>
-                    <div>Previous page<br>
-                        <a href="@controllers.routes.OpportunityController.showOpportunity(opportunity.id, Some(s.sectionNumber))">
-                        @s.title
-                        </a></div>
+                <div class="float-left previous">
+
+                    <span class="arrow-button"></span>
```

Comment:
find it strange that the arrow is not part of the clickable area of the link. I prefer the gov.uk implemetation that affords a wider click area and better outlining upon clicking, see https://www.gov.uk/care-to-learn The relevant code is:

```

<footer>
  <nav class="pagination" role="navigation" aria-label="Pagination">
    <ul class="group">
          <li class="previous">
            <a title="Navigate to previous part" rel="prev" href="/care-to-learn/overview">
              <span class="pagination-label">Previous </span> <span class="pagination-part-title">Overview</span>
</a>          </li>
          <li class="next">
            <a title="Navigate to next part" rel="next" href="/care-to-learn/eligibility">
              <span class="pagination-label">Next</span> <span class="pagination-part-title">Eligibility</span>
</a>          </li>
    </ul>
  </nav>
</footer>
```